flottform/where-are-you-at/src/lib/api.ts

```diff
@@ -1,6 +1,6 @@
 import { env } from '$env/dynamic/public';
 
 export const sdpExchangeServerBase =
-	env.PUBLIC_FLOTTFORM_SERVER_BASE || 'https://192.168.64.1:5177/flottform';
+	env.PUBLIC_FLOTTFORM_SERVER_BASE || 'https:///192.168.0.21:5177/flottform';
 
-export const clientBase = env.PUBLIC_FLOTTFORM_CLIENT_BASE || 'https://192.168.64.1:5176/now/';
+export const clientBase = env.PUBLIC_FLOTTFORM_CLIENT_BASE || 'https:///192.168.0.21:5176/now/';
```

Comment:
I wouldn't commit this changes since we all have different endpoints. Let's keep them locally. And I think there is one `/` too much 😃